Library/Formula/fakesmtp.rb

```diff
@@ -0,0 +1,53 @@
+class Fakesmtp < Formula
+  desc "Dummy SMTP server with GUI for testing emails in applications easily"
+  homepage "https://nilhcem.github.io/FakeSMTP/"
+  url "https://github.com/Nilhcem/FakeSMTP/archive/v2.0.tar.gz"
+  sha256 "8cb623f32b55f814af43b731432eb14da80d95cb13059c79423265037cda9dfb"
+
+  depends_on :java
+  depends_on "maven" => :build
+
+  def install
+    ENV.java_cache
+    system "mvn", "package", "-Dmaven.test.skip=true"
+    libexec.install "target/fakeSMTP-#{version}.jar"
+    (bin/"fakesmtp").write <<-EOS.undent
+      #!/bin/bash
+
+      args+=()
+
+      for i in "$@"; do
+        case $i in
+          -b|--background) args+=('-Djava.awt.headless=true');;
+        esac
+      done
+
+      exec java ${args[@]} -jar #{libexec}/fakeSMTP-#{version}.jar "$@"
+      EOS
+  end
+
+  test do
+    require "net/smtp"
+
+    pid = fork do
+      exec bin/"fakesmtp", "-b", "-s", "-p2025"
```

Comment:
> But without the java parameter -Djava.awt.headless=true it quickly shows a splashscreen on start. This happens even if the --background flag is set. This prevents fakeSMTP to run on headless servers and would make the test fail on travis ci and jenkins severer.

Did it fail previously? Jenkins can handle non-headless tests. Also, this sounds like a `fakesmtp` bug rather than a Homebrew one.